ambari-server/src/main/java/org/apache/ambari/server/orm/entities/ServiceGroupEntity.java

```diff
@@ -80,6 +84,14 @@
   @OneToMany(mappedBy="serviceGroupDependency")
   private List<ServiceGroupDependencyEntity> dependencies;
 
+  @ElementCollection()
```

Comment:
Formatting

Reply:
Can you explain a little more there? Format what? Thanks.

Reply:
I may have meant the next lines down - seems like a lot of whitespace.

Reply:
I will remove all those whitespace, and put them on the same line.